### PR TITLE
Added Search and GeoCode Api

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

File: loklak.php [hunk 1]
```diff
@@ -55,5 +56,54 @@ public function status() {
...
+	public function search($query, $since=null, $until=null, $from_user=null, $count=null){
+        $this->requestURL = $this->baseUrl . '/api/search.json';
+        $this->query = $query;
+        $this->since = $since;
+        $this->until = $until;
+        $this->from_user = $from_user;
+        $this->count = $count;
+        if($query){
+            $params = array('q'=>$this->query);
+            //$params['query'] = $this->$query;
```
Comment (sudheesh001) [inline loklak.php]: Please delete this line.

Comment (daminisatya) [inline loklak.php]: ok!

File: loklak.php [hunk 2]
```diff
@@ -55,5 +56,54 @@ public function status() {
...
+            if ($since)
+            	$params['q'] = $params['q'] . ' since:'.$this->since;
+         	if ($until)
+         		$params['q'] = $params['q'] . ' until:'.$this->until;
+         	if ($from_user)
+         		$params['q'] = $params['q'] . ' from:'.$this->from_user;
+         	if ($count)
+        		$params['count'] = $this->count;
```
Comment (sudheesh001) [inline loklak.php:78]: Indentations of `if ($since)` etc.., are not proper, these are separate if statements and shouldn't be nested.

Comment (daminisatya) [inline loklak.php:78]: ok!

**Comment by sudheesh001** [comment]
@daminisatya looks good now. Thanks 😃

**Pull request merged**